MoreLinq.Test/TriplewiseTest.cs

```diff
@@ -0,0 +1,49 @@
+#region License and Terms
+// MoreLINQ - Extensions to LINQ to Objects
+// Copyright (c) 2008-2011 Jonathan Skeet. All rights reserved.
```

Comment:
Don't be shy, put your name here.

Reply:
Hahah, oops. I just copy-n-pasted the copyright block without paying any attention to it :)